docs/events/AnalysisRunStateChange/examples/ARSC__example1.json

```diff
@@ -20,12 +22,28 @@
     "libraries": [
       {
         "libraryId": "L000001",
-        "orcabusId": "lib.01J5M2J44HFJ9424G7074NKTGN"
+        "orcabusId": "lib.01J5M2J44HFJ9424G7074NKTGN",
+        "readsets": [
+          {
+            "orcabusId": "<readset id> fqr.76J5N2J83RED7387G9374NGDBE",
+            "rgid": "CAGCAGTC+ACGCCAAC.4.990720_A00130_0266_BH7TVMDSX7",
+            "phenotype??": "normal"
```

Comment:
Perhaps we should remove the phenotype...

Reply:
Yes it should